<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>226</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QComboBox" name="SearchLimit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>25</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索页显示条目：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>评论页显示条目：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CommentLimit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>25</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>在线播放品质：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Quality">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>55</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>低</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="Cache">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;当勾选积极缓存时，播放任何歌曲时播放器都会在后台缓存这首歌，这可能导致更高的流量消耗，但如果是使用有线网或Wifi则建议勾选&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;当勾选积极缓存时，播放任何歌曲时播放器都会在后台缓存这首歌，这可能导致更高的流量消耗，但如果是使用有线网或Wifi则建议勾选&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>积极缓存：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>缓存路径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CachePath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowserCache">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Quality_Download">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>85</y>
     <width>69</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>低</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>下载品质：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DownloadPath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowserDownload">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>下载路径：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
